### ISSUE TITLE
Viveport Infinity

### ISSUE THREAD

**Issue by kurohuku7** [alike03/SubscriptionInfo#23]
Could I search for titles from the Viveport Infinity subscription service for VR games?
https://www.viveport.com/infinity

e.g.
https://store.steampowered.com/app/846470/Moss/
https://www.viveport.com/apps/c3212c58-5f00-44a4-aa09-dfd70499253c

I forked this repository and tried to add Viveport myself but it seems to need to change the server side on aligueler.com. I'll appreciate it if it supports Viveport Infinity in the future.

**Comment by alike03** [comment]
Yeah, the data is coming from my site, generated via page scrapers and put together into a database. It's quite messy and requires frequent manual readjustments. Therefore, I'm not interested in increasing that workload.
The only way in adding new platforms I see is, to create all the scraping work much better and as open source with contributes and not just me.
